types/jsreport-core/index.d.ts

```diff
@@ -0,0 +1,98 @@
+// Type definitions for jsreport-core 1.5
+// Project: http://jsreport.net
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 2.6
+
+/// <reference types="node" />
+
+import { Buffer } from 'buffer';
+import { Readable } from 'stream';
+
+type helpers = (...args: any[]) => any;
```

Comment:
Just inline this: `{ [fun: string]: (...args: any[]) => any }`.

Reply:
Will this do?
```ts
type helpers = string | { [fun: string]: (...args: any[]) => any };

interface RenderOptions {
	template: {
		content: string;
		engine: 'jsrender' | 'handlebars' | 'ejs' | 'jade' | string;
		helpers?: helpers;
		recipe: 'phantom-pdf' | 'electron-pdf' | 'text' | 'xlsx' | 'html-to-xlsx' | 'phantom-image' | 'html-to-text' | 'fop-pdf' | 'client-html' | 'wrapped-html' | 'wkhtmltopdf' | string;
	};
	data?: any;
}
```

Reply:
I still question why `helpers` needs a type alias if it's only used in one place. You could just write `helpers?: string | { [fun: string]: (...args: any[]) => any };`